src/main/java/seedu/happypills/HappyPills.java

```diff
@@ -36,12 +36,13 @@ public static void main(String[] args) {
     private void run() {
         ui.printWelcomeMessage();
         Scanner in = new Scanner(System.in);
-        while (true) {
+        while (in.hasNextLine()) {
```

Comment:
This change seem possibly related to your I/O test failure, based on the error message.
You can also compare your approach here with another team that has their current CI passing https://github.com/nus-cs2113-AY1920S2/tp/pulls